src/main/java/com/jadc/bazaar/service/AccountServiceImpl.java

```diff
@@ -0,0 +1,94 @@
+package com.jadc.bazaar.service;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.UUID;
+
+import org.passay.CharacterRule;
+import org.passay.EnglishCharacterData;
+import org.passay.PasswordGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.jadc.bazaar.entity.Account;
+import com.jadc.bazaar.exception.UserNotFoundException;
+import com.jadc.bazaar.repository.AccountRepository;
+import com.mongodb.BasicDBObject;
+
+@Service
+public class AccountServiceImpl implements AccountService {
+	private final AccountRepository accountRepository;
+
+	@Autowired
+	public AccountServiceImpl(AccountRepository accountRepository) {
+		this.accountRepository = accountRepository;
+	}
+
+	@Override
+	public Account save(Account account) {
+		return accountRepository.save(account);
+	}
+
+	@Override
+	public Account findById(Integer id) {
+		Account account = accountRepository.findById(id)
+				.orElseThrow(() -> new UserNotFoundException("No user id " + id + " found"));
+
+		return account;
+	}
+
+	@Override public void insertData(Integer id, String username, String password, String url, String dbName) {
+		Account account = accountRepository.findById(id).get();
+		if(account != null){
+			account.setDbName(dbName);
+			account.setDbUrl(url.concat(dbName));
+			account.setDbUser(username);
+			account.setDbPass(password);
+			account.setUpdatedAt(LocalDateTime.now());
+
+			accountRepository.save(account);
+		}
+	}
+
+	@Override
+	public BasicDBObject createUserWithRole(String role, String dbName, String username, String password) {
+		BasicDBObject user = new BasicDBObject("createUser", username)
+				.append("pwd", password)
+				.append("roles",
+						Collections.singletonList(
+								new BasicDBObject(
+										"role", role).append("db", dbName)
+						));
+		return user;
+	}
+
+	@Override
+	public Account createCredentials(Integer id) {
+		Account findAccount = accountRepository.findById(id).get();
+		Account account = new Account();
+
+		// Generate uuid for username and dbname
+		UUID uuid = UUID.randomUUID();
+		String uuidAsString = uuid.toString();
+
+		// Generate random password and encrypt
+		CharacterRule digits = new CharacterRule(EnglishCharacterData.Digit);
+		CharacterRule lowerCase = new CharacterRule(EnglishCharacterData.LowerCase);
+		CharacterRule upperCase = new CharacterRule(EnglishCharacterData.UpperCase);
+
+		PasswordGenerator passwordGenerator = new PasswordGenerator();
+		String password = passwordGenerator.generatePassword(8, digits, lowerCase, upperCase);
+		System.out.println("Password: "+password);
```

Comment:
I tried now logger.debug using slf4j however I encountered a problem on which it didn't read my `logger.properties` in the resources/dev. So my workaround for this is I directly added the log configuration in my `application-dev.properties` . Do you have an idea Sir on why it didn't read the `logger.properties` file? 
Based on the spring profile, it should have read since it is under the same directory of `application-dev.properties` .

<img width="353" alt="image" src="https://user-images.githubusercontent.com/54928770/192463400-f8c86c15-527c-4aa4-b98d-2dca4845510f.png">

<img width="353" alt="image" src="https://user-images.githubusercontent.com/54928770/192463447-96eb16a3-68d4-49e6-bb6f-afc213918674.png">